force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +225,14 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
-
+        List<Integer> intList = new List<Integer> {1, 2, 3};
         // Add the integers 1, 2, and 3 to the list
         // add 1
         // add 2
         // add 3
 
         // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        return intList; // Replace null with the variable you used to store the result
```

Comment:
Good work using the List constructor. If all you are going to do with a value is return it. I recommend skipping the variable assignment. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
 ```